Exercise-1/Audio-img-video.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Image, Audio and Video</title>
+    </head>
+    <body>
+        <div id="image">               
+            <figure>
+                <img src="lion.jpg" alt="Young lion on a rock" width="642">
```

Comment:
Is not a good practice to use attributes to style stuff. Please use CSS for that